model/event/host_event_finder.go

```diff
@@ -85,3 +86,15 @@ func AllRecentHostEventsMatchStatus(ctx context.Context, hostId string, n int, s
 	return true
 
 }
+
+// CountFailedExecuteEvents counts the number of failed execute script events
+// for a host
+func CountFailedExecuteEvents(id string) (int, error) {
+	filter := ResourceTypeKeyIs(ResourceTypeHost)
+	filter[ResourceIdKey] = id
+	filter[TypeKey] = EventHostScriptExecuteFailed
```

Comment:
It seems like we would want this to only look through the N most recent events for the host to look for repeated failures, right? Otherwise, a long-lived static host could accumulate these events over time and will always return lots of events.